packages/typespec-test/test/openai_generic/generated/typespec-ts/review/openai-generic.api.md

```diff
@@ -417,13 +503,41 @@ export interface FineTuningJobEvent {
     object: string;
 }
 
+// @public (undocumented)
+export interface FineTuningJobsOperations {
+    // (undocumented)
+    jobs: {
+        create: (job: CreateFineTuningJobRequest, options?: CreateOptions_4) => Promise<FineTuningJob>;
+        list: (options?: ListOptions) => Promise<ListPaginatedFineTuningJobsResponse>;
+        retrieve: (fineTuningJobId: string, options?: RetrieveOptions) => Promise<FineTuningJob>;
+        listEvents: (fineTuningJobId: string, options?: ListEventsOptions) => Promise<ListFineTuningJobEventsResponse>;
+        cancel: (fineTuningJobId: string, options?: CancelOptions) => Promise<FineTuningJob>;
+    };
+}
+
+// @public (undocumented)
+export interface FineTuningOperations {
+    // (undocumented)
+    jobs: FineTuningJobsOperations;
+}
+
 // @public
 interface Image_2 {
     b64Json?: Uint8Array;
     url?: string;
 }
 export { Image_2 as Image }
 
+// @public (undocumented)
+export interface ImagesOperations {
+    // (undocumented)
+    images: {
+        create: (image: CreateImageRequest, options?: CreateOptions_10) => Promise<ImagesResponse>;
```

Comment:
I wonder if it's okay to have this CreateOptions_10 here ?

Reply:
I think this is just how API extractor works in order to be able to show everything in the same file (*.api.md) with no conflicts. The actual API that is exported won't have _10